base/combinatorics.jl

```diff
@@ -111,6 +111,23 @@ to verify that `p` is a permutation.
 
 To return a new permutation, use `v[p]`. Note that this is generally faster than
 `permute!(v,p)` for large vectors.
+
+See also [`ipermute!`](@ref)
+
+```jldoctest
+julia> A = [1, 1, 3, 4];
+
+julia> perm = [2, 4, 3, 1];
+
+julia> permute!(A, perm);
```

Comment:
Yeah, I thought it was too verbose to show the return AND that A has been updated.